[Bug]: gnunet build failed

### Problem description


Cloning into '/home/builder/.termux-build/gnunet/cache/tmp-checkout'...
remote: Enumerating objects: 2571, done.
remote: Counting objects: 100% (2571/2571), done.
remote: Compressing objects: 100% (2303/2303), done.
remote: Total 2571 (delta 557), reused 876 (delta 207), pack-reused 0
Receiving objects: 100% (2571/2571), 17.96 MiB | 6.17 MiB/s, done.
Resolving deltas: 100% (557/557), done.
Note: switching to 'b24e01725eb2ea595542b83db2b85cedda72447b'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

Updating files: 100% (2475/2475), done.
Submodule 'contrib/gana' (git://git.gnunet.org/gana.git) registered for path 'contrib/gana'
Cloning into '/home/builder/.termux-build/gnunet/cache/tmp-checkout/contrib/gana'...
Submodule path 'contrib/gana': checked out '6b889c206c1948cf7180e9d5478fd8fba65617ec'
Applying patch: no-librt.patch
Removing folder 'libltdl'...
Submodule path 'contrib/gana': checked out 'dfcc0d45fa75b4e12415eb9ce76d68ccff9fe0da'
Synchronizing submodule url for 'contrib/gana'
recfix --check registry.rec
../format.sh h.template < registry.rec > gnu_name_system_record_types.h.tmp
cat h.header gnu_name_system_record_types.h.tmp h.footer > gnu_name_system_record_types.h
make: *** No rule to make target 'rst.header', needed by 'gnu_name_system_record_types.rst'.  Stop.

[gnunet.txt](https://github.com/termux/termux-packages/files/9390974/gnunet.txt)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/packages/gnunet/no-git.patch
+++ b/packages/gnunet/no-git.patch
@@ -1,0 +1,10 @@
+--- a/bootstrap
++++ b/bootstrap
+@@ -133,7 +133,6 @@
+ main()
+ {
+     cleanup
+-    submodules
+     gana_update
+     check_uncrustify
+     check_yapf

